kyuubi-server/pom.xml

```diff
@@ -427,6 +432,12 @@
             <artifactId>bcpkix-jdk15on</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
do we plan to add a new module for mysql protocol ? or just mix in kyuubi-server ?

Reply:
I think things are quite complex than a new module. we also need to refactor thrift-related classes to make them more abstracted but prefer to do that refactor after adding functionality.

Reply:
The current draft implantation is mixed in `kyuubi-server`, you can get it at #1334.